tests/components/controlBar-test.js

```diff
@@ -926,6 +926,12 @@ describe('ControlBar', function() {
       var wrapperPause = Enzyme.mount(getControlBar(CONSTANTS.STATE.PAUSE));
       expect(typeof wrapperPause.ref('volumeIcon')).toBe('undefined');
     });
+
+    it('uses oo-flex-end class if audio-only', function () {
```

Comment:
Why does this test pass if you have not set an OS to iOS?

Reply:
It was set at the 902 line.